gossip/src/gossip_service.rs

```diff
@@ -71,14 +72,16 @@ impl GossipService {
             None,
             false,
         );
-        let (consume_sender, listen_receiver) = bounded(GOSSIP_CHANNEL_CAPACITY);
+        let (consume_sender, listen_receiver) =
+            EvictingSender::new_bounded(GOSSIP_CHANNEL_CAPACITY);
```

Comment:
Shouldn't the bound here be larger? This channel receives flattened Protocol
values, so intuitively it should be around CAPACITY * BATCH_SIZE?

Reply:
Yep you're right. Since GOSSIP_CHANNEL_CAPACITY isn't a large value, I think we can probably leave it at that. I don't think we need to fine tune these constant too much, as long as we don't accidentally drop more than intended, "larger within limits" is fine imo.